core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -94,19 +104,34 @@ public void start()
         loginAndConsentServer.start();
         URI loginAndConsentBaseUrl = loginAndConsentServer.getBaseUrl();
 
+        certDir = new File(Resources.getResource("cert/localhost.pem").getPath()).getParentFile().toPath();
+        // generate a cert with Docker's host in SAN
+        String ipAddress = DockerClientFactory.instance().dockerHostIpAddress();
+        if (!ipAddress.equals("localhost")) {
```

Comment:
We should remove pem file from resources and always generate it.

Reply:
This file is used in a few other places, not related to this module at all. There's no need to generate it when running with a local Docker daemon, so in 99% of cases.